flink-table-store-core/src/main/java/org/apache/flink/table/store/file/format/FileFormat.java

```diff
@@ -53,10 +53,20 @@
  */
 public abstract class FileFormat {
 
+    protected String formatIdentifier;
+
+    protected FileFormat(String formatIdentifier) {
+        this.formatIdentifier = formatIdentifier;
+    }
+
     protected abstract BulkDecodingFormat<RowData> getDecodingFormat();
 
     protected abstract EncodingFormat<BulkWriter.Factory<RowData>> getEncodingFormat();
 
+    public String getSuffix() {
```

Comment:
Can we unify the naming? I see a lot of places where it's `formatIdentifier`, but here it's `getSuffix`.
We can use `getFormatIdentifier` here.